### ISSUE TITLE
ATC update?

### ISSUE THREAD

**Issue by curki** [epha/robot#11]
http://download.epha.ch/data/atc/atc.json has atc for azilsartan and chlorthalidon

according to bag / swissmedic the code for the combination is C09DA09.

Can this be integrated in the atc.json data??

**Comment by ephaee** [comment]
yes jonas already started https://github.com/epha/robot/commit/e723edac55a89883c3c309b01296d8bbde7851d7

**Comment by ephaee** [comment]
Newest kompendium shows "Edarbyclor, Filmtabletten" with substance "Chlortalidon" for C09DA09.
There is a different product for "Azilsartan" which is "Edarbi, Tabletten" with substance "Azilsartan medoxomil" and ATC C09CA09

https://kompendium.epha.ch/#/gtins=62158,63145/

Is this correct or do we have to corrcet the ATC for this?
